docs/usage.rst

```diff
@@ -4,32 +4,40 @@
 Usage
 =====
 
-Create a :class:`~parsel.selector.Selector` object for the HTML or XML text
-that you want to parse::
+Create a :class:`~parsel.selector.Selector` object for the input text that you want to parse and use.
 
-    >>> from parsel import Selector
-    >>> text = "<html><body><h1>Hello, Parsel!</h1></body></html>"
-    >>> selector = Selector(text=text)
-
-Then use `CSS`_ or `XPath`_ expressions to select elements::
+For HTML or XML, use `CSS`_ or `XPath`_ expressions to select elements::
 
-    >>> selector.css('h1')
+    >>> from parsel import Selector
+    >>> html_text = "<html><body><h1>Hello, Parsel!</h1></body></html>"
+    >>> html_selector = Selector(text=html_text)
+    >>> html_selector.css('h1')
     [<Selector xpath='descendant-or-self::h1' data='<h1>Hello, Parsel!</h1>'>]
-    >>> selector.xpath('//h1')  # the same, but now with XPath
+    >>> html_selector.xpath('//h1')  # the same, but now with XPath
     [<Selector xpath='//h1' data='<h1>Hello, Parsel!</h1>'>]
 
+For JSON format text, use `JMESPath`_ expressions to select fields inside the text::
+
+    >>> json_text = '{"title":"Hello, Parsel!"}'
+    >>> json_selector = Selector(text=json_text)
+    >>> json_selector.jmespath('title')
+    [<Selector xpath='title' data='Hello, Parsel!'>]
```

Comment:
Hmm… Is this the actual output? Maybe we need to fix this. While it is OK for CSS expressions (since they are converted to XPath), here we should probably print `jmespath='title'` instead.

Reply:
Yes, the selector displays xpath='title'. Tested now with an only-json selector.

Reply:
How do you feel about looking into fixing that in code?

Reply:
I looked at it and was thinking of giving my assessment here over the __str__ of the Selector. 
It feels like the problem could be easily fixed by changing the` self.type `to json during a succesful `.jmespath` query. Nevertheless I think this shouldn't be the approach.
The proper way to do it seems to be guaranteeing the `self.type` setup will recognize json, but I stopped there, I felt maybe I was overstepping and would just hinder the progress of the docs.
So yeah if you are opening the possibility for me to fix it then yeah sure I will look further!

Reply:
Maybe there is another way… What if we make it so that, instead of `xpath`, it says `query`?

Reply:
What I am suggesting is to change `xpath` to `query` in `__str__` for every case, regardless of the `type`.

Reply:
Oh, yeah in that case it simplifies a lot the code. Let's go with that approach!